macos/tasks/main.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+# Tasks to be run to provision the macOS buildfarm
+
+# Homebrew stuff
+
+# We redirect stdin from /dev/null so it won't prompt for install.
+#- 
+#  name: "Installing Homebrew"
+#  shell: ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)" < /dev/null
+# NOTE: Might be best to use https://github.com/geerlingguy/ansible-role-homebrew to avoid prompts etc.
+
+
+- 
+  name: "Installing Homebrew packages"
+  homebrew: 
+    name: "{{ item.name }}"
+  with_items: "{{ brew_packages }}"
+
+# Ruby stuff
+
+- 
+  name: "Install RVM prerequisites"
+  # Use get_url here
+  command: curl -sSL "https://rvm.io/mpapis.asc | gpg --import -"
+
```

Comment:
Lets create a files directory in this role and package all these required files there and we can copy them across using the [copy module](http://docs.ansible.com/ansible/copy_module.html) then before running them

We my have issues retrieving files from the web, especially if proxies are installed so this would be best practice I think